README.md

```diff
@@ -1,40 +1,68 @@
 # Pipecat Client Web Transports
 
+[![Docs](https://img.shields.io/badge/Documentation-blue)](https://docs.pipecat.ai/client/reference/js/transports/transport)
+[![Discord](https://img.shields.io/discord/1239284677165056021)](https://discord.gg/pipecat)
+
 A mono-repo to house the various supported Transport options to be used with the pipecat-client-web library. Currently, there is a single transport: daily-transport.
```

Comment:
There are more transports now, right?

Also, maybe `daily-transport` instead of daily-transport?